web-app/src/components/BookingModal/ModalStatusBanner/index.jsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
Just a small thing that if your extracting React already could you just use React.Component to make the class

Reply:
This way of importing Component is generally the best practice.